src/sql/pg/create.sql

```diff
@@ -0,0 +1,147 @@
+    "CASE_STATUS_ID"             INTEGER NOT NULL REFERENCES "N_LU_CASE_STATUS"("CASE_STATUS_ID") ,
+    "EXECUTION_CONDITION_TYPE"   CHARACTER VARYING(20) NULL ,
+    "EXECUTION_CONDITION_DIGEST" CHARACTER VARYING(4000) NULL ,
+    "EXEC_COND_FULFILLMENT"      CHARACTER VARYING(4000) NULL ,
+    "EXPIRES_DTTM"               TIMESTAMP WITH TIME ZONE NULL ,
+    "NOTARIES"                   CHARACTER VARYING(4000) NULL ,
+    "NOTIFICATION_TARGETS"       CHARACTER VARYING(4000) NULL ,
+    "DB_CREATED_DTTM"            TIMESTAMP WITH TIME ZONE DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_DTTM"            TIMESTAMP WITH TIME ZONE DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_USER"            CHARACTER VARYING(40) DEFAULT  CURRENT_USER  NOT NULL 
+);
+
+CREATE INDEX "N_XPK_CASES" ON "N_CASES"
+("CASE_ID"   DESC);
+
+ALTER TABLE ONLY "N_CASES"
+    ADD CONSTRAINT "N_PK_CASES" PRIMARY KEY ("CASE_ID");
+
+CREATE UNIQUE INDEX "N_XAK1CASES_GUID" ON "N_CASES"
+("CASE_UUID"   ASC);
+
+CREATE INDEX "N_XIF1_CASES" ON "N_CASES"
+("CASE_STATUS_ID"   ASC);
+
+CREATE TABLE "N_LU_NOTIFICATION_STATUS"
+(
+    "NOTIFICATION_STATUS_ID"     INTEGER NOT NULL ,
+    "NOTIFICATION_STATUS_CODE"   CHARACTER VARYING(20) NOT NULL ,
+    "NOTIFICATION_STATUS_DESC"   CHARACTER VARYING(255) NULL ,
+    "DB_CREATED_DTTM"            TIMESTAMP WITH TIME ZONE  DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_DTTM"            TIMESTAMP WITH TIME ZONE  DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_USER"            CHARACTER VARYING(40) DEFAULT  USER  NOT NULL 
+);
+
+CREATE UNIQUE INDEX "N_XPKLU_NOTIFICATION_STATUS" ON "N_LU_NOTIFICATION_STATUS"
+("NOTIFICATION_STATUS_ID"   ASC);
+
+ALTER TABLE "N_LU_NOTIFICATION_STATUS"
+    ADD CONSTRAINT  "N_PK_LU_NOTIFICATION_STATE" PRIMARY KEY ("NOTIFICATION_STATUS_ID");
+
+CREATE UNIQUE INDEX "N_XAK1LU_NOTIFICATION_STATUS" ON "N_LU_NOTIFICATION_STATUS"
+("NOTIFICATION_STATUS_CODE"   ASC);
+
+CREATE TABLE "N_NOTIFICATIONS"
+(
+    "NOTIFICATION_ID"            INTEGER NOT NULL DEFAULT nextval('"N_NOTIFICATION_ID_SEQ"'::regclass) ,
+    "NOTIFICATION_STATUS_ID"     INTEGER NOT NULL REFERENCES "N_LU_NOTIFICATION_STATUS"("NOTIFICATION_STATUS_ID"),
+    "CASE_ID"                    INTEGER NOT NULL REFERENCES "N_CASES"("CASE_ID") ,
+    "NOTIFICATION_TARGET"        CHARACTER VARYING(4000) NULL ,
+    "RETRY_COUNT"                INTEGER NULL ,
+    "NEXT_RETRY_DTTM"            TIMESTAMP WITH TIME ZONE NULL ,
+    "IS_ACTIVE"                  SMALLINT NULL ,
+    "LAST_RETRY_DTTM"            TIMESTAMP WITH TIME ZONE NULL ,
+    "DB_CREATED_DTTM"            TIMESTAMP WITH TIME ZONE  DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_DTTM"            TIMESTAMP WITH TIME ZONE  DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_USER"            CHARACTER VARYING(40) DEFAULT  USER  NOT NULL 
+);
+
+CREATE INDEX "N_XPK_NOTIFICATIONS" ON "N_NOTIFICATIONS"
+("NOTIFICATION_ID"   DESC);
+
+CREATE INDEX "N_XIF1_NOTIFICATIONS" ON "N_NOTIFICATIONS"
```

Comment:
In this case I think we were gonna do something like "N_XIF_NOTIFICATIONS_CASES" and "N_XIF_NOTIFICATIONS_STATUS"